saleor/order/models.py

```diff
@@ -164,6 +164,10 @@ def create_history_entry(self, comment='', status=None, user=None):
             status = self.status
         self.history.create(status=status, comment=comment, user=user)
 
+    def create_order_note(self, note='', user=None):
```

Comment:
I think this logic is simple enough to be moved directly to the `create_order` function. We try to keep the models simple and avoid additional methods.